appyters/Gene_Expression_by_Tissue/Gene_Expression_by_Tissue.ipynb

```diff
@@ -164,26 +190,22 @@
     "\n",
     "## Load in normal gene expression\n",
     "Utilize processed datasets containing gene expression by cell type and tissue from [GTEx](https://gtexportal.org/home/) [1], \n",
-    "[ARCHS4](https://maayanlab.cloud/archs4/) [2], and the [Tabula Sapiens](https://tabula-sapiens-portal.ds.czbiohub.org/) [3].\n",
-    "\n",
-    "\n"
+    "[ARCHS4](https://maayanlab.cloud/archs4/) [2], and the [Tabula Sapiens](https://tabula-sapiens-portal.ds.czbiohub.org/) [3]."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "cb3d6c4e",
+   "id": "0dd43774",
    "metadata": {},
    "outputs": [],
    "source": [
     "%%appyter code_exec\n",
     "\n",
     "{% if gtex_gene.value %}\n",
-    "df_bg_stats_gtex = pd.read_csv(\"s3://storage/Tumor_Gene_Target_Screener/gtex-gene-stats.tsv\", storage_options=dict(client_kwargs=dict(endpoint_url=\"https://appyters.maayanlab.cloud\"), anon=True), sep='\\t', index_col=[0,1])\n",
-    "df_bg_genes_gtex = df_bg_stats_gtex.unstack().index.map(lambda idx: lookup(idx.partition('.')[0]))\n",
-    "df_bg_stats_gtex = df_bg_stats_gtex.unstack().groupby(df_bg_genes_gtex, observed=True).median().stack()\n",
-    "df_bg_expr_gtex = df_bg_stats_gtex.loc[(slice(None), ['25%', '50%', '75%']), :].unstack()\n",
-    "{% endif %}\n"
+    "chunks = pd.read_csv(\"s3://storage/Tumor_Gene_Target_Screener/gtex-gene-stats.tsv\", chunksize=100, storage_options=dict(client_kwargs=dict(endpoint_url=\"https://appyters.maayanlab.cloud\"), anon=True), sep='\\t', index_col=[0])\n",
```

Comment:
Neat, I never knew about this chunksize parameter -- i would have just used python's `csv` but using pandas is nice